templates/root/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Community Solid Server</title>
+  <link rel="stylesheet" href="/.well_known/css/styles/main.css" type="text/css">
+</head>
+<body>
+  <header>
+    <img src="/.well_known/css/images/solid.svg" alt="[Solid logo]" />
+    <h1>Community Solid Server</h1>
+  </header>
+  <main>
+    <h1>Welcome to the Community Solid Server</h1>
+    <p>
+      This server implements
+      the <a href="https://solid.github.io/specification/protocol">Solid protocol</a>,
+      so you can set your own Solid pods.
+    </p>
+    <p>
+      Some things you can try:
+    </p>
+    <ul>
+      <li>
+        Read or write documents in the <strong>default pod</strong>.
+        <ul>
+          <li>This is the pod you are reading from now.</li>
+          <li>Anyone can read or write;
+              update the <a href="/.acl">access controls</a> to change this.</li>
+          <li>You can disable the default pod in the configuration.</li>
```

Comment:
To be adjusted according to reality then? Perhaps easily:

```suggestion
          <li>You can make changes to the default pod in the configuration.</li>
```